src/io/flutter/settings/FlutterSettings.java

```diff
@@ -27,6 +27,7 @@
   private static final String showOnlyWidgetsKey = "io.flutter.showOnlyWidgets";
   private static final String syncAndroidLibrariesKey = "io.flutter.syncAndroidLibraries";
   private static final String showStructuredErrors = "io.flutter.showStructuredErrors";
+  private static final String includeAllStackTraces = "io.flutter.ncludeAllStackTraces";
```

Comment:
nit: can we make this member on the one on line 29 use the word "Key" for consistency?

Reply:
I should at least change the spelling of ".ncl" to ".incl" but there are others that do not include "Key". I'll just change them all.